File: C#/E1.AddTests/CharacterizationTests1.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace TDDLegacyCode.E1.AddTests
{
    [TestClass]
    public class CharacterizationTests1
    {
        [TestMethod]
        public void CalculatorTests()
        {
            Calculator calc = new Calculator();
            Assert.AreEqual("0", calc.GetDisplay());

            calc.Press("3");
            Assert.AreEqual("3", calc.GetDisplay());
        }
    }
}

File: TS/TDD-Legacy-Code/E1.AddTests/CharacterizationTests1.ts
import assert = require('assert');
import { Calculator } from './Calculator';

describe('characterization test', function () {

    it('calculator tests', function () {
        let calc = new Calculator();
        assert.equal("0", calc.getDisplay());

        calc.press("3");
        assert.equal("3", calc.getDisplay());
    });
});

